data/rbac_approval_seed.yml

```diff
@@ -0,0 +1,85 @@
+---
+resource_definition1: &AdminResourceDefinition
+  attributeFilter: 
+    value: '*'
+    key: id
+    operation: equal
+access1: &AdminReadAccessTemplates
+  permission: approval:templates:read
+  resourceDefinitions:
+          - *AdminResourceDefinition
+access2: &AdminCreateAccessWorkflows
+  permission: approval:workflows:create
+  resourceDefinitions:
+          - *AdminResourceDefinition
+access3: &AdminReadAccessWorkflows
+  permission: approval:workflows:read
+  resourceDefinitions:
+          - *AdminResourceDefinition
+access4: &AdminUpdateAccessWorkflows
+  permission: approval:workflows:update
+  resourceDefinitions:
+          - *AdminResourceDefinition
+access5: &AdminReadAccessRequests
+  permission: approval:requests:read
+  resourceDefinitions:
+          - *AdminResourceDefinition
+access6: &AdminReadAccessStages
+  permission: approval:stages:read
+  resourceDefinitions:
+          - *AdminResourceDefinition
+access7: &AdminReadAccessActions
+  permission: approval:actions:read
+  resourceDefinitions:
+          - *AdminResourceDefinition
+access8: &AdminCreateAccessActions
+  permission: approval:actions:create
```

Comment:
@hsong-rh Should this be write instead of create, or do you want to keep create and update separate.

Reply:
Yes, we want to keep the difference between `create` and `update`. This is also the reason why we don't use `write`. So far `update` is only allowed on `workflow` for admin.